### ISSUE TITLE
Unclear how to start the visual IDE

### ISSUE THREAD

**Issue by davidpomerenke** [enso-org/enso#3354]
I would like to start Enso's visual IDE. I already succeeded at this a few months ago, but now I am very confused how I should do it:

- The [two](https://github.com/enso-org/ide/releases) [links](https://github.com/enso-org/enso/tree/develop/gui) from the repo README are outdated.
- Anyway, I checked the releases. The situation could be clearer, cf. https://github.com/enso-org/enso/issues/3213. I went ahead and downloaded the enso bundle, and successfully installed it, nice! The installation seems to contain the launcher and the engine, as far as I understand. I suppose this is the "non-visual" part of enso.
- Somewhere in the documentation it is mentioned that one needs the project manager for running the visual IDE. It seems not to be included in the bundle, so I downloaded it. Running it, I get `Started server at 127.0.0.1:30535, press enter to kill server`. I opened the site in my browser, giving: `Expected WebSocket Upgrade request`. So it seems to me that it should be used by some other component of enso, and not on its own.

**TLDR**: It's very unclear how to start the visual IDE. Updating the README / documentation in this regard would be grea ☺️

**Comment by wdanilo** [comment]
@davidpomerenke I'm really sorry to hear you had such a bad experience, and you are right, the docs needs to be more clear. We are right now working on improving our release process and CI config. As soon as it will be done, we will rework our docs.

**Comment by Igorgro** [comment]
@wdanilo any update on this?

**Comment by IsaacTell** [comment]
I think the download experience is now much better. https://github.com/enso-org/enso/blob/develop/README.md now points to https://github.com/enso-org/enso/releases which has clear instructions of what to download. But on windows at least the install seems to complete silently with no indication if it was successful or what to launch. (It was successful and I just had to find enso in start menu)

**Comment by farmaazon** [comment]
> I think the download experience is now much better. https://github.com/enso-org/enso/blob/develop/README.md now points to https://github.com/enso-org/enso/releases which has clear instructions of what to download. But on windows at least the install seems to complete silently with no indication if it was successful or what to launch. (It was successful and I just had to find enso in start menu)

The windows' installation should be now fixed: there's a progress bar, and the application just launches once finished.

<blockquote><img src="https://opengraph.githubassets.com/3ad7cddb28c444dddf5c1d6d5ba92be3cfae430b5df3f88500bb8595eb74f5b4/enso-org/enso" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/enso-org/enso/blob/develop/README.md">enso/README.md at develop · enso-org/enso</a></strong></div><div>Enso Analytics is a self-service data prep and analysis platform designed for data teams. - enso-org/enso</div></blockquote>
<blockquote><img src="https://opengraph.githubassets.com/3ad7cddb28c444dddf5c1d6d5ba92be3cfae430b5df3f88500bb8595eb74f5b4/enso-org/enso" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="/enso-org/enso/releases">Releases · enso-org/enso</a></strong></div><div>Enso Analytics is a self-service data prep and analysis platform designed for data teams. - enso-org/enso</div></blockquote>
